scene/gui/color_picker.cpp

```diff
@@ -48,6 +48,13 @@ void ColorPicker::_notification(int p_what) {
 			btn_pick->set_icon(get_theme_icon("screen_picker", "ColorPicker"));
 			bt_add_preset->set_icon(get_theme_icon("add_preset"));
 
+			uv_edit->set_custom_minimum_size(Size2(get_theme_constant("sv_width"), get_theme_constant("sv_height")));
+			w_edit->set_custom_minimum_size(Size2(get_theme_constant("h_width"), 0));
+			for (int i = 0; i < 4; i++) {
+				labels[i]->set_custom_minimum_size(Size2(get_theme_constant("label_width"), 0));
+				set_margin((Margin)i, has_theme_constant_override("margin") ? get_theme_constant("margin") : get_margin(static_cast<Margin>(i)));
```

Comment:
Why do we check for the local override? The constant is more likely applied via the theme and this condition would ignore that.